README.md

```diff
@@ -8,9 +8,118 @@ Ballerina Constraint Library
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-constraint.svg?label=Last%20Commit)](https://github.com/ballerina-platform/module-ballerina-constraint/commits/master)
   [![GitHub issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/constraint.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fconstraint)
 
-This library provides features to validate the values that have been assigned to Ballerina types.
-
-The Ballerina `constraint` library facilitates APIs to do validations on the Ballerina types further with the use of annotations.
+The Ballerina `constraint` library provides annotations to add constraints to Ballerina types and API to validate the values with respect to the constraints defined in the respective types.
```

Comment:
```suggestion
The Ballerina `constraint` library provides annotations to add constraints to Ballerina types and an API to validate the values with respect to the constraints defined in the respective types.
```

or

```suggestion
The Ballerina `constraint` library provides annotations to add constraints to Ballerina types and a function to validate the values with respect to the constraints defined in the respective types.
```

Reply:
Using API rather than function

Fixed via [9c58851](https://github.com/ballerina-platform/module-ballerina-constraint/pull/123/commits/9c58851bfdca933e13b09490135ee6b9b64025de)